crates/vsock/src/vhu_vsock_thread.rs

```diff
@@ -126,7 +136,39 @@ impl VhostUserVsockThread {
                 ),
             );
         }
-
+        let (sender, receiver) = mpsc::channel::<EventData>();
+        thread::spawn(move || loop {
+            let event_data = receiver.recv().unwrap();
```

Comment:
This will take down the process if `-C panic=abort` is set when the sender goes away.

Perhaps
`let Ok(event_data) = receiver.recv() else break`?

Reply:
done!